### ISSUE TITLE
Design Key metrics Setup CTA

### ISSUE THREAD

**Issue by jimmymadon** [google/site-kit-wp#6210]
## Feature Description

The Key metrics Setup CTA placeholder component created in #6209 should be designed fully in this issue.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* The Key metrics Setup CTA component created in #6209 should be designed as per the [Figma mock](https://www.figma.com/file/CdtrHB0HAjfGRWTnTMMQI0/SummaryWidget?node-id=1292%3A34229&t=CCVl3OjC54mlpgYR-0).
* Clicking on the _Get tailored metrics_ Primary Button should begin the User Input v2 questionnaire.
* Clicking on the Secondary button and Footer links should do nothing as part of this issue.

## Implementation Brief

- Wait until #6209 is merged.
- We will replace the placeholder widget created in #6209 (mentioned in its ACs):
    - Note that the component will no longer be in use on `SettingsAdmin` once #6718 is merged. 
        - So, it makes sense to give it a more context-aware name `KeyMetricsCTA` instead of the current `UserInputSettings`
        - remove `isDismissible` prop. 
        - See this comment [thread](https://github.com/google/site-kit-wp/pull/6718#discussion_r1202199664) for more context on it. 
    - Start off with a `<section>` element having the class `.googlesitekit-plugin .googlesitekit-setup__wrapper`.
	- Inside the `<section>` component, construct a `<Grid />` component having two `<Row />` components, one for content and the other for the footer `Maybe later` action.
	    - In the first `<Row />` component, place two `<Cell />` components, each having half width of their container.
		    - The first `<Cell />` component should have the text content and the actions (`Get tailored metrics` `<Button />` and `I'll pick metrics myself` `<Link />`). Style the elements matching the Figma design.
			    - The `Get tailored metrics` `<Button />` component should take the user to the User Input questionnaire. The URL for it can be obtained using the `getAdminURL` selector of the `core/site` store, passing `googlesitekit-user-input` to it.
				- The `I'll pick metrics myself` `<Link />` component should do nothing at this point.
			- Export the `Ghost cards` element from the Figma design, and place it in the second `<Cell />` component. To ensure this SVG placement is graceful, you can follow the implementation in `assets/js/modules/idea-hub/components/common/IdeaHubPromptSVG.js`.
		- In the second `<Row />` component, place the `Maybe later` `<Link />` component inside a full-width `<Cell />`  component. It should do nothing at this point, its action will be implemented as a part of #6232.
- Add a Storybook story for the CTA widget. 

### Test Coverage

- Add a new test suite for the CTA widget, where test cases can be added gradually. Start by adding a test case for clicking on the `Get tailored metrics` `<Button />` component.

## QA Brief

* Enable the `userInput` feature flag and set up Site Kit.
* Don't answer User Input Questions.
* Ensure the GA4 module is connected.
* Go to the Site Kit dashboard; ensure the old User Input Settings banner notification doesn't appear.
* Ensure new Key Metrics Setup CTA appears on the dashboard as per the Figma designs.
* Ensure that clicking the `Get tailored metrics` button navigates the user to the user input questionnaire screen.

**Verify the layout in the following viewports**

**Upto 600px**

<img width="572" alt="Screenshot 2023-07-03 at 10 58 52 AM" src="https://github.com/google/site-kit-wp/assets/24408261/5ef6f153-3b53-4acd-bef5-b6f67d719a7b">

-----

**601px - 960px**

<img width="584" alt="Screenshot 2023-07-03 at 10 58 32 AM" src="https://github.com/google/site-kit-wp/assets/24408261/72187020-38c2-402f-9fc0-d3115fd5becc">

<img width="781" alt="Screenshot 2023-07-03 at 11 01 31 AM" src="https://github.com/google/site-kit-wp/assets/24408261/776d7500-14de-4c60-b32d-612dce307328">

------

**961px - 1280px**

<img width="736" alt="Screenshot 2023-07-03 at 10 59 42 AM" src="https://github.com/google/site-kit-wp/assets/24408261/8c773eb0-e6a6-4e80-82f0-42e19df7feda">

------

**1281px and above**

<img width="902" alt="Screenshot 2023-07-03 at 11 00 01 AM" src="https://github.com/google/site-kit-wp/assets/24408261/aa215169-ec69-4047-87cd-f92cd4d2bbe3">

## Changelog entry

* Implement the design for the Key Metrics Setup CTA widget.

**Comment by tofumatt** [comment]
IB ✅

**Comment by mohitwp** [comment]
### QA Update ⚠️ 

@hussain-t 

- Tested on dev environment.
- Verified KM set up CTA with Figma.
- Verified CTA in different view ports as mentioned under Figma.
- Verified that Clicking on the Get tailored metrics Primary Button begin the User Input v2 questionnaire.
- Note : Clicking on the Secondary button and Footer links should do nothing as part of this issue.

**Question 1>** 
CTA title and content font and font size is different from Figma. It is similar to other CTA banner but different from Figma. Is this intentional for maintaining consistency ?

<details>

![image](https://github.com/google/site-kit-wp/assets/94359491/68b7e54c-8806-41c9-9c45-b3fffe6dd324)

![image](https://github.com/google/site-kit-wp/assets/94359491/43e4badf-0175-42aa-9267-1b0aba37c8a3)

![image](https://github.com/google/site-kit-wp/assets/94359491/8cbea9cf-14f4-45c5-85e3-10268aef4722)

![image](https://github.com/google/site-kit-wp/assets/94359491/23568581-e9a3-433e-a716-a987ff9ad8f5)

</details>

**Question 2>** In view ports having width between 601- 960 px space is inconsistent and lot of space appears below buttons. Can we do some improvement in mobile design ? Should I create a new ticket for this ?

![image](https://github.com/google/site-kit-wp/assets/94359491/f7edf5a5-66b6-4cca-9fc1-2b2dcfa5984f)

**Comment by nfmohit** [comment]
@mohitwp Just FYI, for **Question 1**, [we have noticed](https://github.com/google/site-kit-wp/pull/7255#discussion_r1253515036) these differences and included a fix for them in the PR for #6265. Thanks!

CC: @hussain-t

**Comment by hussain-t** [comment]
Thanks, @mohitwp. Regarding your questions:

>CTA title and content font and font size is different from Figma. It is similar to other CTA banner but different from Figma. Is this intentional for maintaining consistency ?

Yes, this is expected to be consistent with other banners. It will be fixed in #6265 as @nfmohit mentioned.


>Question 2> In view ports having width between 601- 960 px space is inconsistent and lot of space appears below buttons. Can we do some improvement in mobile design ? Should I create a new ticket for this ?

This is due to the SVG image height. I will create a follow-up PR to fix it shortly. We don't need to create a new ticket.

**Comment by mohitwp** [comment]
### QA Update ✅ 

- Tested on dev environment.
- Verified KM set up CTA with Figma.
- Verified CTA in different view ports as mentioned under Figma.
- Verified that Clicking on the Get tailored metrics Primary Button begin the User Input v2 questionnaire.
- Note : 
Clicking on the Secondary button and Footer links should do nothing as part of this issue.
Figma's difference reported [above](https://github.com/google/site-kit-wp/issues/6210#issuecomment-1623557028) will be fix through #6265

<details>

![image](https://github.com/google/site-kit-wp/assets/94359491/4790cfbc-5488-4aab-bcfa-97cd0253d4a2)

![image](https://github.com/google/site-kit-wp/assets/94359491/07fc2278-0bef-4bf7-b78a-530410f2828f)

![image](https://github.com/google/site-kit-wp/assets/94359491/4905fcbb-eacd-4b0b-a021-a4481cfc4b92)

![image](https://github.com/google/site-kit-wp/assets/94359491/14169418-1e52-4143-8395-6b11a46857e6)

![image](https://github.com/google/site-kit-wp/assets/94359491/833b6e0e-42a6-4981-9b3c-7a2560bc26d6)

![image](https://github.com/google/site-kit-wp/assets/94359491/761990da-1703-4818-8834-a07e6ada70a8)

</details>
